### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Akumetsu-Dev** [AtomLinter/linter-csslint#153]
```
Error: Column start (2) greater than line length (1)
    at Object.rangeFromLineNumber (C:\Users\Walddys Emmanuel.atom\packages\linter-csslint\node_modules\atom-linter\lib\index.js:179:11)
    at C:/Users/Walddys Emmanuel/.atom/packages/linter-csslint/lib/main.js:88:30
    at Array.forEach (native)
    at C:/Users/Walddys Emmanuel/.atom/packages/linter-csslint/lib/main.js:74:31
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

start today doing this, happend after save proyect.

**Comment by Arcanemagus** [comment]
It's extremely likely that csslint is reporting an invalid point back to us, if you can post the file at the state where you get this error I can verify whether this is the case.

Unfortunately with the current state of CSSLint, it's unlikely anyone will actually fix this. I would _highly_ recommend you look into switching to [stylelint](http://stylelint.io/), of which [linter-stylelint](https://atom.io/packages/linter-stylelint) is the Atom package.

**Comment by Akumetsu-Dev** [comment]
body, a, ul, div, table, td, tr, img, li{
  margin: 0;
  padding: 0;
  text-decoration: none;
}
/********************************************* Banner_Begin *********************************************/
.tbanner{
  min-width: 100%;
  height: 100px;
  background-color: #0EEB88;
}
.imagen{
  background-color: #20AFCF;
  height: 100px;
}
.imagen img{
  height: 100px;
  background-color: #2037CF;
}
.textban{
  z-index: 2;
  width: 385px;
  background-color: #EB89DD;
  position: relative;
  float: right;
  height: 30px;
  top: -105px;
  text-align: right;
  font: normal 52px Cherry Swash;
}
/********************************************* Banner_End *********************************************/

/********************************************* DropMenu_Begin *********************************************/
.menu{
  margin: 0;
  z-index: 1;
  min-height: 50px;
  background: #1A2649;
  border: 1px solid #561388;
}

.dropdown{
  position: relative;
  display: block;
  min-width: 100px;
  height: 50px;
  float: left;
  border-right: 1px solid red;
}

.dropbtn{
  float: left;
  display: block;
  text-align: center;
  width: 100px;
  height: 50px;
  line-height: 50px;
  background: #E5E5E5;
}

.dropContent{
  border-bottom-left-radius: 10px;
  border-bottom-right-radius: 10px;
  visibility: hidden;
  /* This hides the menu until we hover */
  position: absolute;
  top: 50px;
  max-height: 150px;
  overflow: hidden;
}

.dropdown:hover .dropContent{
  /* This makes the menu visible when user hovers */
  visibility: visible;
  text-align: center;
}

.dropContent a{
  color: #6A101B;
  background: #476262;
  display: block;
  width: 125px;
  height: 30px;
  text-align: center;
  line-height: 175%;
}

.dropContent a:hover{
  background: #447D7D;
  color: #FFFFFF;
}

.menudrop{
  visibility: visible;
  opacity: 0;
  height: 0;
          transition: opacity 0.25s ease-in-out, height 0.25s ease-in-out;
       -o-transition: opacity 0.25s ease-in-out, height 0.25s ease-in-out;
      -ms-transition: opacity 0.25s ease-in-out, height 0.25s ease-in-out;
     -moz-transition: opacity 0.25s ease-in-out, height 0.25s ease-in-out;
  -webkit-transition: opacity 0.25s ease-in-out, height 0.25s ease-in-out;
}

.dropdown:hover .menudrop{
  opacity: 1;
  height: 90px;
}

/********************************** DropMenu_End **********************************/
/********************************** SlideShow_Begin **********************************/
.slide{
  display: none;
}
.slideshow{
  margin: auto;
  max-width: 900px;
  position: relative;
  z-index: -1;
}
.slide img{
  width: 100%;
  height: 225px;
}
.numeroslide{
  color: #B1ABAB;
  font-size: 12px;
  padding: 4px 12px;
  position: absolute;
  top: 0;
}
.punto{
  position: relative;
  top: -25px;
  height: 13px;
  width: 13px;
  margin: 0 2px;
  background-color: #BBBBBB;
  border-radius: 50%;
  display: inline-block;
  -webkit-transition: background-color 0.6s ease;
     -moz-transition: background-color 0.6s ease;
      -ms-transition: background-color 0.6s ease;
       -o-transition: background-color 0.6s ease;
          transition: background-color 0.6s ease;
}

.active{
  background-color: #717171;
}

.fade{
  -webkit-animation-name: fade;
  -webkit-animation-duration: 1.5s;
     -moz-animation-name: fade;
     -moz-animation-duration: 1.5s;
       -o-animation-name: fade;
       -o-animation-duration: 1.5s;
      -ms-animation-name: fade;
      -ms-animation-duration: 1.5s;
          animation-name: fade;
          animation-duration: 1.5s;
}

@-webkit-keyframes fade{
  from{
    opacity: 0.4;
  }
  to{
    opacity: 1;
  }
}

@keyframes fade{
  from{
    opacity: 0.4;
  }
  to{
    opacity: 1;
  }
}
/********************************** SlideShow_End **********************************/
/********************************** ModalBox_Begin **********************************/
/* The Modal (background) */
.modalbox{
  z-index: 10;
  display: none; /* Hidden by default */
  position: fixed; /* Stay in place */
  padding-top: 180px; /* Location of the box */
  left: 0;
  top: 0;
  min-width: 100%; /* Full width */
  min-height: 100%; /* Full height */
  overflow: auto; /* Enable scroll if needed */
  background-color: rgb(0,0,0); /* Fallback color */
  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
}
/* Modal Content */
.modalcont{
  position: relative;
  background-color: #FEFEFE;
  margin: auto;
  padding: 0;
  border: 1px solid #888888;
  max-width: 80%;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
  -webkit-animation-name: animatetop;
  -webkit-animation-duration: 0.4s;
     -moz-animation-name: animatetop;
     -moz-animation-duration: 0.4s;
      -ms-animation-name: animatetop;
      -ms-animation-duration: 0.4s;
       -o-animation-name: animatetop;
       -o-animation-duration: 0.4s;
          animation-name: animatetop;
          animation-duration: 0.4s;
}
/* Add Animation */
@-webkit-keyframes animatetop{
  from{
    top: -300px;
    opacity: 0;
  }
  to{
    top: 0;
    opacity: 1;
  }
}

@keyframes animatetop{
  from{
    top: -300px;
    opacity: 0;
  }
  to{
    top: 0;
    opacity: 1;
  }
}

/* The Close Button */
.close{
  color: white;
  float: right;
  font-size: 28px;
  font-weight: bold;
}

.close:hover, .close:focus{
  color: #000000;
  text-decoration: none;
  cursor: pointer;
}

.modal-header{
  padding: 2px 16px;
  background-color: #5CB85C;
  color: white;
}

.modal-body{
  padding: 2px 16px;
}

.modal-footer{
  padding: 2px 16px;
  background-color: #5CB85C;
  color: white;
}
/********************************** ModalBox_End **********************************/
/********************************** Mobile_Or_Resize_Screen_Begin **********************************/
@media only screen and (max-width:720px){
  .menu li{
    display: none;
  }
  .textban{
    z-index: 2;
    width: 250px;
    background-color: #EB89DD;
    position: relative;
    float: right;
    top: -67px;
    text-align: right;
    font: normal 33px Cherry Swash;
  }
}
/********************************** Mobile_Or_Resize_Screen_End **********************************/

**Comment by Arcanemagus** [comment]
Marking this as closed, if you are interested in getting this fixed please file an issue over on [CSSLint](https://github.com/CSSLint/csslint/issues).
